src/third_party/dartino/object_memory.cc

```diff
@@ -132,7 +132,15 @@ void Space::iterate_objects(HeapObjectVisitor* visitor, LivenessOracle* filter)
         ASSERT(size > 0);
         current += size;
       } else {
-        current += object->size(program_);
+        word size = object->size(program_);
+#ifdef DEBUG
```

Comment:
Should we stop eagerly initializing arrays and instances?

Reply:
Let's let this cook for a week, then remove the eager initialization.